src/tucal/plugins/__init__.py

```diff
@@ -2,9 +2,11 @@
 import typing
 
 import tucal.plugins.c187B12
+import tucal.plugins.eventHTU
```

Comment:
Please change the name of your file to `htu_events` and the name of the Plugin class to `HTUEvents`